cloudos/clos.py

```diff
@@ -699,3 +699,28 @@ def workflow_import(self, workspace_id, workflow_url, workflow_name,
             raise BadRequestException(r)
         content = json.loads(r.content)
         return content['_id']
+
+    def get_user_info(self, verify=True):
+        """Gets user information from users/me endpoint
```

Comment:
What is a "me" endpoint?

Reply:
`v1/users/me` is the api endpoint to get user info. It is used to inspect if the user has set docker.io credentials or not